libs/langchain/langchain/callbacks/__init__.py

```diff
@@ -79,4 +79,5 @@ def __getattr__(name: str) -> Any:
     "SageMakerCallbackHandler",
     "LabelStudioCallbackHandler",
     "TrubricsCallbackHandler",
+    "UpTrainCallbackHanlder",
```

Comment:
We are in the process of breaking dependency of langchain on community, so we don't want anyone to be importing community based code from `langchain`

```suggestion
```